modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= {oadp-full} support
```

Comment:
I recommend moving the word "support" to the front of the heading to make it easier to read because the `{oadp-full}` attribute translates into a five-word noun phrase. I mean something like this:
```suggestion
= Support for {oadp-full}
```

Reply:
Something outside of the scope of this PR:
I have a similar suggestion about the heading that precedes this one:
https://82722--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-intro#oadp-apis_oadp-api
![image](https://github.com/user-attachments/assets/00ac37aa-9c77-41ae-a238-6144b54a6a3e)
It is better read as `APIs for {oadp-full}` because the `{oadp-full}` attribute translates into a five-word noun phrase.